AdminWebsite/AdminWebsite/ClientApp/src/app/booking/search-email/search-email.component.ts

```diff
@@ -42,6 +42,8 @@ export class SearchEmailComponent implements OnInit, OnDestroy {
 
     @Input() initialValue = '';
 
+    @Input() locator = 'participantEmail';
```

Comment:
I prefer hyphens for ids. I don't think we have any rule for this and I'm ok with it being different when it comes from code.

Any thought for making the prefix an input and having a static value for the end?

Reply:
I see now the previous value was participantEmail. That's fine, I'm guessing that some acceptance tests rely on that.

Reply:
yeah few automation tests are still depending on that